src/main/io/osd.c

```diff
@@ -659,410 +456,47 @@ void osdUpdateAlarms(void)
     int32_t alt = osdGetAltitude(BaroAlt) / 100;
     statRssi = rssi * 100 / 1024;
 
-    if (statRssi < OSD_cfg.rssi_alarm)
-        OSD_cfg.item_pos[OSD_RSSI_VALUE] |= BLINK_FLAG;
+    if (statRssi < masterConfig.osdProfile.rssi_alarm)
```

Comment:
Just cosmetics, but getting and using a reference to masterConfig.osdProfile would make this code a bit more compact.